pkg/roachprod/roachprod.go

```diff
@@ -739,6 +739,18 @@ func Stop(ctx context.Context, l *logger.Logger, clusterName string, opts StopOp
 	return c.Stop(ctx, l, opts.Sig, opts.Wait, opts.MaxWait)
 }
 
+// Signal sends a signal to nodes in the cluster.
```

Comment:
We already have the option of passing a signal to `Stop`[^1] and I believe this works with any signal, so we should either live with that (my suggestion) or try to improve the APIs around all of this (sounds like a distraction). Or is there a reason why you needed to add this?

[^1]: https://github.com/cockroachdb/cockroach/blob/87510fd6444c74c19b1f854f548a91afbb9d9025/pkg/roachprod/roachprod.go#L711

Reply:
Just that stop is meant to, well, stop cluster nodes, and so it e.g. outputs and logs messages about stopping when `SIGCONT` will in fact start the nodes.

Reply:
I don't really want to drag you into a cleanup since it'll touch a lot of surface, but for roachtest/prod I think it would be better to standardize around a shared implementation of `Signal`, rather than introducing two ways of sending signals that are basically interchangeable except that they log differently.

I don't want to block this merge, my preference would be to introduce Signal as a pure alias to Stop and perhaps edit some of the log messages in the shared implementation to be a bit more signally and less stoppy, and in lieu of that just use `Stop` here with a comment+issue link, but if you feel strongly you can go ahead and merge.

Reply:
Pulled the implementation out to a `kill()` helper, passing through any relevant output args.